.circleci/config.yml

```diff
@@ -76,6 +77,11 @@ jobs:
       SETTINGS__solr__username: catalog
       SETTINGS__solr__password: catalog
     steps:
+    # - run:
```

Comment:
Do you think we'll turn uncomment this at any point? If not, we could just remove it.

Reply:
I think we should remove this.

Reply:
Good catch guys. That was leftover from some troubleshooting